ui/containers/GameLobbyContainer.jsx

```diff
@@ -1,12 +1,17 @@
 import { TimeSync } from "meteor/mizzao:timesync";
 import { withTracker } from "meteor/react-meteor-data";
+import React from "react";
 import moment from "moment";
 
 import { LobbyConfigs } from "../../api/lobby-configs/lobby-configs.js";
-import GameLobby from "../components/GameLobby.jsx";
 
-// Handles all the timing stuff
-export default withTracker(({ gameLobby, player, ...rest }) => {
+export default class GameLobbyContainer extends React.Component {
```

Comment:
I don't think this GameLobbyContainer is necessary.

Simply export withTracker (without calling it on anything):

```jsx
export const GameLobbyContainer = withTracker(({ ... }) => { ... });
```

Then in Game.jsx, just calling on the component. 

```jsx
return GameLobbyContainer(Lobby || DelayedGameLobby)
```
withTracker() returns a higher-order component (https://reactjs.org/docs/higher-order-components.html), which you can compose this way by just calling it on a component that needs it. We make most our containers by default attached to a single component, but it doesn't have to be that way. Naming wise, calling it a Container while it a higher-order component is not ideal, but it's not a huge deal either. ;)

Reply:
the reason why i did that is to prevent the auto rendering, even though i refactor the `TimeSync.serverTime` to `Tracker.Dependency` the auto rendering still happen ( i also tried to commented it out).

i think we also cannot do the  `return GameLobbyContainer(Lobby || DelayedGameLobby)` we need pass it as a component e.g., ` <GameLobbyContainer />`. beside that, we also need to call the `withTracker` function first put it inside a variable, and then we call that variable as a component. i tried it but the result the components was always rendering( i commented the `TimeSync.serverTime`)

so far, i am still looking the more proper way to do this.

Reply:
Ok. I will take a look at this. Thank you Putu.